latest/ug/clusters/windows-support.adoc

```diff
@@ -72,7 +78,8 @@ aws iam attach-role-policy \
   --role-name eksClusterRole \
   --policy-arn {arn-aws}iam::aws:policy/AmazonEKSVPCResourceController
 ----
-. Create a file named [.replaceable]`vpc-resource-controller-configmap.yaml` with the following contents.
+. If the VPC CNI *is not* installed as an Amazon EKS Add-on:
```

Comment:
VPC CNI is is always installed for networking in k8s. This command just enable Windows-ipam inside VPC resource controller.

Rephrase the statement to, enable IPAM for Windows nodes in VPC resource controller using following ConfigMap.

Reply:
CNI for Windows is different from Linux. Windows CNI is shipped in AMIs. It is not installed via addon.

Reply:
ah tyty